app/views/posts/index.html.erb

```diff
@@ -1,4 +1,22 @@
-<h1>All the posts from a user</h1>
-<h3>Post number 1</h3>
-<h3>Post number 2</h3>
-<h3>Post number 3</h3>
+<% @users = User.all; %>
+<% current_user = @users.find_by(id: params[:user_id]) %>
+<div class="body">
+  <h2>All the posts from <%= current_user.name %></h2>
+  <div>
+    <%= render partial: "users/user", locals: { current_user: current_user } %>
+  </div>
+  <% @posts_per_user = @users.find_by(id: params[:user_id]).posts %>
+  <% @posts_per_user.first(3).each do |post| %>
+    <% @comments_per_post = Post.find_by(id: post.id).comments %>
+    <%= render partial: "users/post", locals: { post: post, current_user: current_user } %>
+    <h3>Comments:</h3>
+    <div class="comments">
+      <% @comments_per_post.first(5).each do |comment| %>
```

Comment:
- [ ] please avoid using logic in the views, you previously created a method called `last_5_comments` in your Post model to handle this.